### PR TITLE
Judgment localizations and more

### PR DESCRIPTION
**Pull request by LordLink78**
also added and changed splashes

### CODE REVIEW

File: _ark/ui/locale/esl/locale_updates_keep.dta [hunk 1]
```diff
@@ -393,8 +393,14 @@
 (os_highway_username_desc "Muestra el Usuario en la Pista")
 (os_highway_overdrive "Sobrecarga")
 (os_highway_overdrive_desc "Muestra el porcentaje de Sobrecarga en la Pista")
-(os_highway_streak "Racha")
+(os_highway_streak "Racha")
```
Comment (carlmylo) [inline _ark/ui/locale/esl/locale_updates_keep.dta]: Extra space at the end. Remove.

File: _ark/ui/locale/esl/locale_updates_keep.dta [hunk 2]
```diff
@@ -422,6 +428,22 @@
...
+(os_perfects_indidicator_on_desc "Mostrar presición perfecta")
+(os_perfects_indidicator_off "Perfecto: No")
+(os_perfects_indidicator_off_desc "No mostrar precisión perfecta")
+
+(os_detailed_hit_on "Mas Detalles: Si")
+(os_detailed_hit_on_desc "Mostrar notificaciones de Perfecto, Bien, OK, Mal")
+(os_detailed_hit_off "Mas detalles: No")
+(os_detailed_hit_off_desc "No mostrar notificaciones de Perfecto, Bien, OK, Mal")
+
+(os_hit_timing_on "Proximinad: Si")
```
Comment (carlmylo) [inline _ark/ui/locale/esl/locale_updates_keep.dta]: `Proximinad` should be `Proximidad`

File: _ark/ui/locale/esl/locale_updates_keep.dta [hunk 3]
```diff
@@ -422,6 +428,22 @@
 (os_highway_streak_gh3_delay_on_desc "No mostrará la racha individual hasta que se llegue a una racha de 25 notas")
 (os_highway_streak_gh3_delay_off "Racha GH3: No")
 (os_highway_streak_gh3_delay_off_desc "Mostrará la racha individual desde la primer nota")
+
+(os_perfects_indidicator_on "Perfecto: Si")
+(os_perfects_indidicator_on_desc "Mostrar presición perfecta")
```
Comment (carlmylo) [inline _ark/ui/locale/esl/locale_updates_keep.dta]: `presición` should be `precisión`

File: _ark/ui/locale/esl/locale_updates_keep.dta [hunk 4]
```diff
@@ -422,6 +428,22 @@
 (os_highway_streak_gh3_delay_on_desc "No mostrará la racha individual hasta que se llegue a una racha de 25 notas")
 (os_highway_streak_gh3_delay_off "Racha GH3: No")
 (os_highway_streak_gh3_delay_off_desc "Mostrará la racha individual desde la primer nota")
+
```
Comment (carlmylo) [inline _ark/ui/locale/esl/locale_updates_keep.dta:431]: Error in the original file. @jnackmclain should correct this after this is merged.
Line 432 to 435 say `indidicator` but it should be `indicator`

**Review by carlmylo — CHANGES_REQUESTED** [review]
From @Lynxeption

File: _ark/ui/locale/esl/locale_updates_keep.dta [hunk 5]
```diff
@@ -395,6 +395,12 @@
 (os_highway_overdrive_desc "Muestra el porcentaje de Sobrecarga en la Pista")
 (os_highway_streak "Racha")
 (os_highway_streak_desc "Muestra la Racha de Notas de un jugador individualmente en la Pista")
+(judge_bad "MAL")
+(judge_good "OK")
+(judge_great "BIEN")
+(judge_perfect "PERFECTO")
```
Comment (Aloquendiar) [inline _ark/ui/locale/esl/locale_updates_keep.dta:401]: Maybe
Bad = Malo
Good = Ok
Great = Genial

**Review by Aloquendiar — COMMENTED** [review]
It may be a little confusing considering that great means you are doing better than good but before perfect (it can stay as it is now, which is also fine)

**Pull request merged**